### PR TITLE
Add arm image support for release

### PR DESCRIPTION
**Pull request by anamnavi**
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.

### CODE REVIEW

File: .vsts-ci/phase.yml [hunk 1]
```diff
@@ -122,6 +122,18 @@ jobs:
...
+      Write-Host "##vso[task.setvariable variable=GOVERNANCE_SYSTEM_DOTNET;]$dotnetFolder"
+    displayName: Set GOVERNANCE_SYSTEM_DOTNET
+    condition: and( succeededOrFailed(), ne(variables['Channel'],''))
+
+  - pwsh: |
+      Get-ChildItem env:\GOVERNANCE*
+    displayName: 'Capture GOVERNANCE* Environment Variables'
+    condition: and( succeededOrFailed(), ne(variables['Channel'],''))
+
+
```
Comment (TravisEz13) [inline .vsts-ci/phase.yml:136]: Is this still needed?

**Review by TravisEz13 — COMMENTED** [review]
Have you tested this change?

File: .vsts-ci/releasePhase.yml [hunk 2]
```diff
@@ -99,15 +104,38 @@ jobs:
...
+        foreach ($module in $modules) {
+          $modulesOnMachine = get-module -listavailable
+          Write-Host $modulesOnMachine -Verbose
+          if (!(Get-Module $module -listavailable)) {
+            Write-Verbose "installing $module..." -verbose
+            Install-Module $module -force -allowclobber
+          } else {
+            Write-Verbose "$module is already installed" -verbose
+          }
+        }
```
Comment (TravisEz13) [inline .vsts-ci/releasePhase.yml:134]: This needs to go into a separate pwsh task before you ever trying to use Azure PowerShell.

Explanation:  You can't user AzurePowerShell if it is not installed yet.

**Comment by dolauli** [comment]
Is there an ETA when the PowerShell ARM image will be released.

File: .vsts-ci/releasebuild.yml [hunk 3]
```diff
@@ -4,9 +4,9 @@ parameters:
   - name: 'previewReleaseTag'
     default: 'v7.4.0-preview.1'
   - name: 'ltsReleaseTag'
-    default: 'v7.2.8'
+    default: 'v7.2.9'
```
Comment (TravisEz13) [inline .vsts-ci/releasebuild.yml:7]:
```suggestion
    default: 'v7.2.10'
```

File: .vsts-ci/releasebuild.yml [hunk 4]
```diff
@@ -4,9 +4,9 @@ parameters:
...
   - name: 'stableReleaseTag'
-    default: 'v7.3.1'
+    default: 'v7.3.2'
```
Comment (TravisEz13) [inline .vsts-ci/releasebuild.yml:9]:
```suggestion
    default: 'v7.3.3'
```

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**